### ISSUE TITLE
Can't download data with python script

### ISSUE THREAD

**Issue by ecbland** [SuperDARNCanada/globus#6]
I'm no longer able to download data using `sync_radar_data_globus.py`. I get the following error: 

```
sync_radar_data_globus.py -y 2018 -m 12 -p 20181210*han .
Listing path: /chroot/sddata/raw/2018/12/ on endpoint: c02cb494-1515-11e9-9f9f-0a06afd4a22e with pattern: name:~*20181210*han*rawacf.bz2
Note: This can take several minutes
.
Transferring 12 files with a soft timeout of 360 s.
Globus API error
Error code: EndpointNotFound
Error message: No such endpoint with legacy name '7379b8f6-c3cb-11eb-87e2-559da91cd9a3
'
```

I attempted to fix the problem by updating `globusconnectpersonal` to the latest version, and also pulling the updates to `sync_radar_data_globus.py` from gihub. However, I still get the same error. 

I can still use the Globus website to download data, so my endpoint does exist. And the legacy name in the error message matches the legacy name of my endpoint.

**Comment by mts299** [comment]
Sent a support ticket in to globus support

**Comment by kevinkrieger** [comment]
@ecbland , can you try changing line 64 of your script from this:
`PERSONAL_UUID = f.readline()`
to this:
`PERSONAL_UUID = f.readline().strip()`

I think the behaviour of the API must have changed on Globus' end, since this affected multiple people across operating systems and devices. readline() seems to add a newline and removing it seems to work for me.

**Comment by kevinkrieger** [comment]
This should be fixed with #7 . Globus support confirmed this was an issue.
